modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/components/MDFClaimPage/schema/yup.ts

```diff
@@ -119,28 +96,26 @@ const claimSchema = object({
 															.requestAmount
 													)
 											)
-											.required(),
+											.required('Required'),
 								}),
 								requestAmount: number(),
 							})
 						),
 				}),
-				listOfQualifiedLeads: mixed().when('selected', {
-					is: (selected: boolean) => selected,
+				eventProgram: mixed().when(['selected', 'typeActivity'], {
+					is: (selected: boolean, typeActivity: LiferayPicklist) =>
+						selected && typeActivity.key === TypeActivityKey.EVENT,
 					then: (schema) =>
 						schema
+							.required('Required')
 							.test(
 								'fileSize',
 								validateDocument.fileSize.message,
-								(listOfQualifiedLeads) => {
-									if (
-										listOfQualifiedLeads &&
-										!listOfQualifiedLeads.id
-									) {
-										return listOfQualifiedLeads
+								(eventProgram) => {
+									if (eventProgram && !eventProgram.id) {
+										return eventProgram
```

Comment:
Improve this logic, the ternary it's not necessary since the same condition is validate in if statement